<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSelfCrossCorrelationClass</class>
 <widget class="QDialog" name="CSelfCrossCorrelationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自相关-互相关</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>输出文件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="OutputFile1Label">
        <property name="text">
         <string>输出周期信号F0：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="OutputFile1PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="OutputFile2Label">
        <property name="text">
         <string>输出非周期信号F1：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="OutputFile2PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="OutputFile3Label">
        <property name="text">
         <string>输出信号F0的自相关：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="OutputFile3PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="OutputFile4Label">
        <property name="text">
         <string>输出信号F1的自相关：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="OutputFile4PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="OutputFile5Label">
        <property name="text">
         <string>输出F0与F1的互相关：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="OutputFile5PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="OutputFile6Label">
        <property name="text">
         <string>输出F1与F0的互相关：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="OutputFile6PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="OutputFile7Label">
        <property name="text">
         <string>输出F0与F1互相关的翻转：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="OutputFile7LineEdit"/>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="OutputFile7PushButton">
        <property name="text">
         <string>. . .</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="OutputFile6LineEdit"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="OutputFile5LineEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="OutputFile4LineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="OutputFile3LineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="OutputFile2LineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="OutputFile1LineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="CalPushButton">
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="CancelPushButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="InputParGroupBox">
     <property name="title">
      <string>输入信号参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="Par1Label">
        <property name="text">
         <string>f0(Hz)：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="Par4DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="Par1DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="Par3DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="Par7DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Par5Label">
        <property name="text">
         <string>N0：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="Par2DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Par3Label">
        <property name="text">
         <string>A0：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="Par4Label">
        <property name="text">
         <string>A1：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Par7Label">
        <property name="text">
         <string>β：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="Par2Label">
        <property name="text">
         <string>f1(Hz)：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="Par5DoubleSpinBox">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="Par8Label">
        <property name="text">
         <string>△t（ms）：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="Par8DoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="Par6Label">
        <property name="text">
         <string>N1：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="Par6DoubleSpinBox">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="self_cross_correlation.qrc"/>
 </resources>
 <connections/>
</ui>
